Gopkg.toml

```diff
@@ -0,0 +1,50 @@
+# Gopkg.toml example
```

Comment:
This is what `dep init` gave me: https://golang.github.io/dep/docs/migrating.html